lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -295,14 +295,23 @@ def perform_module_state_update(update_fcn, filter_fcn, _entry_id, course_id, ta
 
     """
     start_time = time()
-    usage_key = course_id.make_usage_key_from_deprecated_string(task_input.get('problem_url'))
+    usage_keys = []
+    problem_url = task_input.get('problem_url')
+    entrance_exam_url = task_input.get('entrance_exam_url')
```

Comment:
I hate to say this, but I wonder if it is better to model this that the celery task sender creates an array of problem_urls (which are all gradeable elements under the entrance exam) rather than letting the concept of an 'entrance_exam_url' bleed over into the celery task worker.

I say this because it could be useful - in the general case - to allow for multiple problem rescoring to be done. These two use cases (i.e. resccoring all items in an entrance exam and rescoring an aribitrary list of problems) seem the the same to me.

This isn't a must have, just a suggestion.